rpcmocksrv/server.go

```diff
@@ -50,8 +50,8 @@ func New() *Server {
 }
 
 func randomPort() int {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
I don't this know if this needs to be truly random, but we need to keep rand.Seed if it does probably

Reply:
I made this change as the linter had a problem with **weak random** used before, but this is part of `rpcmocksrv` so I wouldn't worry about this